src/pages/about-us.js

```diff
@@ -0,0 +1,18 @@
+import Head from 'components/Head'
+import getAboutUsData from '../../data/about-us'
+import PageContent from 'components/PageContent'
+import AboutUsHero from 'blocks/AboutUsHero'
```

Comment:
I see that in `blocks` there is an `index.js` which means that normally you should be able to do 
```
import { aboutUsHero } from 'blocks'
```

Is there a particular reason why you prefer to go directly to the specific file?

Reply:
No specific reason, I'm just used to do it this way :) Do you want me to update it ?

Reply:
I think it's the purpose of `blocks/index.js`, so it might be better. What do you think?

Reply:
Done ;)